docs/src/index.md

```diff
@@ -26,6 +26,35 @@ in Julia.
     Your changes will be vetted by developers before becoming permanent, so don't
     worry about whether you might say something wrong.
 
-```@contents
-Pages = ["install.md", "quickstart.md", "arrays_colors.md", "conversions_views.md", "indexing.md", "imageaxes.md", "imagefiltering.md", "imagemetadata.md", "imagesegmentation.md", "function_reference.md", "api_comparison.md"]
-```
+The following is a list of packages that are closely related to the JuliaImages ecosystem and also maintained by members of
```

Comment:
I like the idea here, but I worry this list will be intimidating for newcomers ("Welcome to JuliaImages! Here are 60 different packages you have to master. Good luck!"), especially if it appears on the front page. I'd rather start with a gentle introduction before getting into a long list spelling out what code goes where. Let's just do `using Images` until readers gain a bit of experience with JuliaImages.

Reply:
This is moved to the front page of the "Pakcages" part. And I've also added some words to explain how the documentation is organized and built.